include/staff/templates/users.tmpl.php

```diff
@@ -34,9 +34,9 @@
 $page=($_GET['p'] && is_numeric($_GET['p']))?$_GET['p']:1;
 $pageNav=new Pagenate($total,$page,PAGE_LIMIT);
 $qstr = '&amp;'. Http::build_query($qs);
-$qs += array('sort' => $_REQUEST['sort'], 'order' => $_REQUEST['order']);
+$qs += array('id' => $org->getId(), 'sort' => $_REQUEST['sort'], 'order' => $_REQUEST['order']);
 
-$pageNav->setURL('users.php', $qs);
+$pageNav->setURL(basename($_SERVER['PHP_SELF']), $qs);
```

Comment:
I made changes you wrote, but I think it doesn't work fine. The missing part in new URL is "id=<number>", where number is id of organization. 

With your changes the url looks like: ...../scp/orgs.php?sort=&order=&p=2

but it should:

....../scp/orgs.php?id=5&p=2